highfidelity/hifi
# Whack-A-Mole

## Whack-a-Mole game for Marketplace

Whack moles with your friends, using the red and blue mallet.

Internal tracker: https://highfidelity.fogbugz.com/f/cases/859/Whack-a-mole

## QA Test

Try testing this with two persons.

1. Load up the whackAMole script to create it. Edit -> Running Scripts -> From URL ( https://cdn.rawgit.com/thoys/hifi/8c7e3fa0ab141da8941612cfa2e3b921244a2795/unpublishedScripts/marketplace/whackAMole/createWhackAMoleConsole.js )
2. Far trigger the push button.
3. Drop the coin into the slot.
4. The game should start. You have 30 seconds to whack as many moles you can. Each player has their own mallet. (left = blue, right = red).
5. The game should move back to attract mode after the game. There is 20% chance that the mole laughs after emerging from its hole.
